cdap-docs/application-templates/source/etl/templates/sources/real-time/index.rst

```diff
@@ -11,6 +11,5 @@ Sources: Real-time
 
     JMS <jms>
     Kafka <kafka>
-    SQS <sqs>
```

Comment:
Albert got back to me about the deleted file: that was an error; it should have been a rename..we're working on fixing it.